docs/testing-guidelines/WritingPesterTests.md

```diff
@@ -69,81 +55,91 @@ Note that if `Get-Item` were to succeed, the test will fail.
+If the tag is not provided, tests in that describe block will be run any time tests are executed.
 
-#### Describe
+### Describe
 
 Creates a logical group of tests.
 All `Mocks` and `TestDrive` contents defined within a `Describe` block are scoped to that `Describe`;
 they will no longer be present when the `Describe` block exits.
-A `Describe` block may contain any number of Context and `It` blocks.
+A `Describe` block may contain any number of `Context` and `It` blocks.
+
+### Context
+
+Provides logical grouping of `It` blocks within a single `Describe` block. Any `Mocks` defined inside a `Context` are removed at the end of the `Context` scope, as are any files or folders added to the `TestDrive` during the `Context` block's execution. 
 
-#### Context
+Any `BeforeEach` or `AfterEach` blocks defined inside a `Context` also only apply to tests within that `Context`.
 
-Provides logical grouping of `It` blocks within a single `Describe` block. Any Mocks defined inside a `Context` are removed at the end of the `Context` scope, as are any files or folders added to the `TestDrive:` during the Context block's execution. Any `BeforeEach` or `AfterEach` blocks defined inside a `Context` also only apply to tests within that `Context`.
+### It
 
-#### It
+The `It` block is intended to be used inside of a `Describe` or `Context` block. If you are familiar with the AAA pattern (Arrange-Act-Assert), the body of the  `It` block is the appropriate location for an assert.
 
-The  `It` block is intended to be used inside of a `Describe` or `Context` block. If you are familiar with the AAA pattern (Arrange-Act-Assert), the body of the  `It`  block is the appropriate location for an assert. The convention is to assert a single expectation for each `It` block. The code inside of the `It` block should throw a terminating error if the expectation of the test is not met and thus cause the test to fail. The name of the `It` block should expressively state the expectation of the test.
+The convention is to assert a single expectation for each `It` block. The code inside of the `It` block should throw a terminating error if the expectation of the test is not met and thus cause the test to fail.
 
-### Admin privileges in tests
+The name of the `It` block should expressively state the expectation of the test.
+
+## Admin privileges in tests
 
 Tests that require admin privileges **on Windows** must be additionally marked with `RequireAdminOnWindows` Pester tag.
+
 In the Azure DevOps Windows CI, we run two different passes:
 
-- The pass with exclusion of tests with `RequireAdminOnWindows` tag
-- The pass where we run only `RequireAdminOnWindows` tests
+- The pass with exclusion of `RequireAdminOnWindows` tagged tests.
+- The pass where only `RequireAdminOnWindows` tagged tests are being tested.
```

Comment:
"are being tested" -> "are being executed"

Reply:
```suggestion
- The pass where only `RequireAdminOnWindows` tagged tests are being executed.
```

Reply:
Fixed. Thank you!